<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузка профиля пользователя </string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QListWidget" name="PacientSH">
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="selBTN">
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="delBTN">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
